packages/templates/react-urql-graphql-codegen/src/components/App.tsx

```diff
@@ -0,0 +1,20 @@
+import { useGeneratedQuery } from '../generated'
+import Fetching from './Fetching'
+import Error from './Error'
+import ListPlaceholder from './ListPlaceholder'
+
+function App() {
+  const [result] = useGeneratedQuery({
+    variables: { search: "%%" }
```

Comment:
Is or possible `variables: {}`?

Reply:
i changed the query, so it doesn't have to be there  parameter "search"